### DISCUSSION TITLE
S3 storage won't work with an existing bucket.

### DISCUSSION THREAD

**Discussion by RphCos** [requarks/wiki#5461] [Error / Bug Report]
Hello,

First of all thank you for all the work on this amazing project.

I tried using the S3 connector for backup of my wiki but ran into a problem.
I tried to feed the S3 form an already existing bucket without success. I got an infinite loop of "Unexpected error" showing up.  

I would like to be able to use the S3 storage with an account having only List and Read/Write Object permissions on a given bucket, as it is often the case with S3 integrations.

Also the documentation on the S3 connector is absent https://docs.requarks.io/storage/s3, which is why I might have misunderstood what is expected, and I'm sorry if it is the case.

```
OS:  Debian 11 with Docker
Wiki.js version 2.5.283
SQLite Database
```

For anyone else finding this post, the alternative I implemented is simply to add a local storage export for backup, in my case as I run it inside docker it is a bind from the host, and then from the aws console on the host configured with the backup account I simply S3 sync.

**Comment by samjain2907** [comment]
Hi, 
Getting the same issue. I need to use wikijs as a production-grade documentation tool for my organization. I wanted to use S3 for versioning as well as backups. Since there is no support for Gitlab too yet, is there a way for me to work this out? I am also getting an infinite loop of "Unexpected error" showing up upon connection to the S3 bucket.

**Reply by NGPixel** [reply]
GitLab is supported... You can use any service compatible with Git.

**Comment by RajaATAli** [comment]
> Hello,
> 
> First of all thank you for all the work on this amazing project.
> 
> I tried using the S3 connector for backup of my wiki but ran into a problem. I tried to feed the S3 form an already existing bucket without success. I got an infinite loop of "Unexpected error" showing up.
> 
> I would like to be able to use the S3 storage with an account having only List and Read/Write Object permissions on a given bucket, as it is often the case with S3 integrations.
> 
> Also the documentation on the S3 connector is absent https://docs.requarks.io/storage/s3, which is why I might have misunderstood what is expected, and I'm sorry if it is the case.
> 
> ```
> OS:  Debian 11 with Docker
> Wiki.js version 2.5.283
> SQLite Database
> ```
> 
> For anyone else finding this post, the alternative I implemented is simply to add a local storage export for backup, in my case as I run it inside docker it is a bind from the host, and then from the aws console on the host configured with the backup account I simply S3 sync.

Hi,

I am facing the same issue. Were you able to solve it somehow? Thank you :)
